kubectl-plugin/pkg/cmd/get/get.go

```diff
@@ -8,11 +8,12 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-func NewClusterCommand(streams genericclioptions.IOStreams) *cobra.Command {
+func NewGetCommand(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:          "cluster",
-		Short:        "Manage ray cluster resources",
-		Long:         `Allow users to manage and retrieve ray cluster information and resources.`,
+		Use:          "get",
+		Short:        "Display one or many Ray resources.",
```

Comment:
This is technically incorrect since it only supports getting RayCluster right?

Reply:
Won't we support getting RayJob and RayService in the future?

Reply:
ah good point